taxi_service/settings.py

```diff
@@ -122,7 +122,11 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
+
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
```

Comment:
The STATICFILES_DIRS setting should contain a list of locations where Django will look for static files. It's currently set to `BASE_DIR / 'static'`, which means Django will look for static files in a directory named 'static' at the root of your project. Make sure this directory exists and contains your static files.